src/ucp/tag/tag_recv.c

```diff
@@ -105,7 +105,7 @@ ucp_tag_recv_common(ucp_worker_h worker, void *buffer, size_t count,
         }
 
         req->recv.prev_flags          = req->flags;
-        req->flags                    = UCP_REQUEST_FLAG_RECV | req_flags;
+        req->flags                    = UCP_REQUEST_FLAG_RECV;
```

Comment:
no `param` flags is going to be used in the future?

Reply:
in this case no need to set CALLBACK flag because we call it directly, and other `param` flags are not relevant for now

Reply:
yes, but there is a risk you will miss flags from param if they are added later

Reply:
@brminich but these are request flags, it's not the same as param flags, what am I missing here?